### ISSUE TITLE
How to convert the weights trained by darknet?

### ISSUE THREAD

**Issue by yanzi6969** [eric612/MobileNet-YOLO#36]
Thank you for your great work.
   I have a question. I did some changes on  yolov3-tiny  cfg,  trained  on darknet and got a  good result. Now ,I want to make the weights work on the ncnn.Can I convert the weights and cfg by your system directly, or I have to first convert my cfg to caffe format,then train it on your system again?
  Look forward to your reply

**Comment by eric612** [comment]
There are two ways for yolo detection on ncnn : 

* [Directly porting](https://github.com/xiangweizeng/darknet2ncnn)
* Use this caffe pytorch [converter](https://github.com/eric612/MobileNet-YOLO/tree/master/models/darknet_yolov3) 

Unfortunately in case 2 , has a issue was the pooling layer which param set stride = 1 kernel = 2, the output dim were different between darknet and caffe . So I suggest replace the pooling layer with kernel = 3 and pad = 1 and finetune in caffe . 

Here was my finetune prototxt
https://drive.google.com/open?id=1YRHO2w1r-K_QCypn23tQJPZXKW-JzH4m

Currently , I didn't upload letterbox resize code to ncnn , so you need check your model was trained from alexAB or original darknet , due to the issue  
https://github.com/AlexeyAB/darknet/issues/232#issuecomment-336955485
